dashboard/app/controllers/levels_controller.rb

```diff
@@ -158,10 +158,9 @@ def edit
     @in_script = @level.script_levels.any? || any_parent_in_script
     @standalone = ProjectsController::STANDALONE_PROJECTS.values.map {|h| h[:name]}.include?(@level.name)
     if @level.is_a? Applab
-      # TODO: unfirebase, migrate this to datablock storage, ok to be migrated but not enabled: #56998
+      # TODO: unfirebase, this is migrated to datablock storage but NOT TESTED: #56998
       # TODO: post-firebase-cleanup, remove this code: #56994
-      fb = FirebaseHelper.new('shared')
-      @dataset_library_manifest = fb.get_library_manifest
+      @dataset_library_manifest = DatablockStorageLibraryManifest.instance.library_manifest
     end
```

Comment:
How do we test this is working?

Reply:
Tested in level editor that after updating the manifest, we saw the updated manifest in level editor!